src/com/assignment2/KaprekerConstant.java

```diff
@@ -0,0 +1,28 @@
+package com.assignment2;
+
+public class KaprekerConstant {
+
+	public static boolean checkKaprekersConstant(int number) {
+		int ascendingSortedNumber = NumberManipulator.ascendingSort(number);
+		int descendingSortedNumber = NumberManipulator.descendingSort(ascendingSortedNumber);
+
+		int difference;
+		int iterationCount = 0;
+		boolean isKaprekersConstant = false;
+
+		while (iterationCount < Constants.MAX_DIFFERENCE_ITERATION) {
+			difference = Math.abs(ascendingSortedNumber - descendingSortedNumber);

```

Comment:
[diffrence = descendingSortedNumber - ascendingSortedNumber] 
can be used instead of Math.abs function as descendingSortedNumber will always greater than ascendingSortedNumber.